docs/Diagnostics/HealthChecker/NodeRunnerMemoryLimitCheck.md

```diff
@@ -0,0 +1,13 @@
+# Noderunner.exe Memory Limit Check
+
+### Description
+
+This check is looking at the `C:\Program Files\Microsoft\Exchange Server\V15\Bin\Search\Ceres\Runtime\1.0\noderunner.exe.config` to look at the `memoryLimitMegabytes` value. This value should be set to 0 for the best performance. By having it set to 0, we do not limit the `noderunner.exe` processes. However, in some scenarios you might want to recommend to limit the process memory consumption to prevent server impact. If you do this, it is only recommended as a temporary fix.
```

Comment:
```suggestion
This check is looking at the `<ExchangeInstallPath>\Bin\Search\Ceres\Runtime\1.0\noderunner.exe.config` to look at the `memoryLimitMegabytes` value. This value should be set to 0 for the best performance. By having it set to 0, we do not limit the `noderunner.exe` processes. However, in some scenarios you might want to recommend to limit the process memory consumption to prevent server impact. If you do this, it is only recommended as a temporary fix.
```